fineract-provider/src/main/java/org/apache/fineract/commands/api/MakercheckersApiResource.java

```diff
@@ -77,13 +81,10 @@ public List<AuditData> retrieveCommands(@Context final UriInfo uriInfo, @BeanPar
 
```

Comment:
Not sure if this should be secured!?? Usually there is a validation happening here...